index.js

```diff
@@ -8,6 +9,7 @@ const trackedChanges = {
     "update": "!"
 }
 
+const expandDetailsComment = core.getInput('expand-comment');
```

Comment:
```suggestion
const expandDetailsComment = core.getBooleanInput('expand-comment');
```
I think this might be why the condition isn't working like it should

Reply:
Good catch.